tools/pctools/venv_inst.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# COPY and RUN this script in a container image, to:
+# create a python virtualenv named as $1,
+# pip install the modules provided as parameters from $2 on,
+# collect the resulting dependencies in a resource_$1.txt file
+# copy the txt in a common dir.
+# --- 
+# $1 CLI Cloud type: aws, azure, ec2, openstack
+# $2 resources output file
+# $3,... : install modules
+# ---
+
+echo $0 $*
+
+# parameters check
+[ $# -lt 3 ] && exit 1
```

Comment:
Might be a good idea to exit with an error message, WDYT?

```suggestion
[ $# -lt 3 ] && (echo "not enough arguments"; exit 1)
```

Reply:
thanks, indeed I already did so, but the main process of the command RUN resulted still ok and the build doesn't stop, because in this way the eventual error occurs in a sub-shell (...) but the parent shell is ok. 
I should convert the command using a normal 'if ... then'.